tempodb/backend/s3/s3.go

```diff
@@ -648,10 +648,11 @@ func createCore(cfg *Config, hedge bool) (*minio.Core, error) {
 	}
 
 	opts := &minio.Options{
-		Region:    cfg.Region,
-		Secure:    !cfg.Insecure,
-		Creds:     creds,
-		Transport: transport,
+		Region:           cfg.Region,
+		Secure:           !cfg.Insecure,
+		Creds:            creds,
+		Transport:        transport,
+		DualStackEnabled: cfg.UseDualStack,
```

Comment:
This option doesn't exist, it's configured via a method [`SetS3EnableDualstack`](https://github.com/minio/minio-go/blob/14b3aa6d56708fdf2071346aadcdb085069d8937/api.go#L342-L346) on the client (what's returned by `minio.NewCore()`)

Reply:
Sure @mapno , I appreciate your help throughout and would be glad if you're able to find time to fix this. I'll try to learn better and contribute. Thanks